build.gradle

```diff
@@ -69,68 +69,45 @@ import org.gradle.internal.os.OperatingSystem;
 
 tasks.withType(Test) {
     outputs.upToDateWhen { false } // tests will always rerun
-    useTestNG()
+
+    // Always run serially because there are some very badly behaved tests in HTSJDK that
+    // will cause errors and even deadlocks if run multi-threaded
+    maxParallelForks = 1
 
     // set heap size for the test JVM(s)
     minHeapSize = "1G"
     maxHeapSize = "2G"
 
     jvmArgs '-Djava.awt.headless=true'  //this prevents awt from displaying a java icon while the tests are running
 
-    if (System.env.CI == "true") {  //if running under a CI output less into the logs
```

Comment:
I don't disagree @lbergelson, though I've just spent the last two hours trying to make this work and am running out of ideas.  It seems like the combination of gradle + scalatest + TestNG make this rather hard to figure out.  The code that was in the build file previously doesn't seem to suppress much of anything (I changed the test to `if (true)` to test locally).

How critical do you think this is?